mlir-sycl/test/Dialect/SYCL/invalid.mlir

```diff
@@ -613,3 +613,17 @@ gpu.module @kernels {
     gpu.return
   }
 }
+
+// -----
+
+func.func @f(%event: !llvm.ptr, %queue: !llvm.ptr) {
+  // expected-error @below {{'sycl.host.submit' op '@kernels::@k0' does not reference a valid CGF function}}
```

Comment:
What do you mean by 'CGF' here ?

Reply:
CGF is a "command-group function", defined in the SYCL specification [here](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#command-group-function-object) and [here](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:command.group.scope).